chart/values.yaml

```diff
@@ -113,11 +113,11 @@ images:
     pullPolicy: IfNotPresent
   pgbouncer:
     repository: apache/airflow
-    tag: airflow-pgbouncer-2023.02.24-1.16.1
+    tag: airflow-pgbouncer-2024.01.19-1.21.0
```

Comment:
We need to wait with merging until https://github.com/apache/airflow/discussions/36870 brings some thumbs-up by the user.

Reply:
Let's do a little leap of faith here. Tests passed. So things are working.